### PR TITLE
Fixes visible message when showing paper

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Swaps the `visible_message()` arguments around for showing someone a piece of paper, since they were backwards.
This meant that you'd get the `"You show the paper to [person]."` message when you're not the one doing it.

Also fixed wiping your face with paper not removing lipstick.

## Why It's Good For The Game

Bugfix.

## Changelog
:cl:
fix: Fixed the 'visible' and 'user' message being the wrong way round when showing a paper to someone.
fix: Fixed not being able to wipe lipstick off with paper.
/:cl:

### CODE REVIEW

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -135,29 +135,28 @@
...
-		if(!istype(M, /mob))	return
-
-		if(ishuman(M))
-			var/mob/living/carbon/human/H = M
-			if(H == user)
-				to_chat(user, "<span class='notice'>You wipe off your face with [src].</span>")
+		if(H == user)
+			to_chat(user, "<span class='notice'>You wipe off your face with [src].</span>")
+			H.lip_style = null
+			H.update_body()
```
Comment (farie82) [inline code/modules/paperwork/paper.dm:151]:
```suggestion
```

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -135,29 +135,28 @@
...
+		else
+			user.visible_message("<span class='warning'>[user] begins to wipe [H]'s face clean with \the [src].</span>",
+							 	 "<span class='notice'>You begin to wipe off [H]'s face.</span>")
+			if(do_after(user, 1 SECONDS, target = H) && do_after(H, 1 SECONDS, FALSE))	//user needs to keep their active hand, H does not.
+				user.visible_message("<span class='notice'>[user] wipes [H]'s face clean with \the [src].</span>",
+					"<span class='notice'>You wipe off [H]'s face.</span>")
 				H.lip_style = null
 				H.update_body()
```
Comment (farie82) [inline code/modules/paperwork/paper.dm:159]:
```suggestion
			if(!do_after(user, 1 SECONDS, target = H) && do_after(H, 1 SECONDS, FALSE))	//user needs to keep their active hand, H does not.
				return
			user.visible_message("<span class='notice'>[user] wipes [H]'s face clean with \the [src].</span>",
		H.lip_style = null
		H.update_body()
```
Combine this with the above. This way you avoid dupe code for cleaning the lip style etc

File: code/modules/paperwork/paper.dm [hunk 3]
```diff
@@ -135,29 +135,28 @@
...
+/obj/item/paper/attack(mob/living/carbon/M, mob/living/carbon/user, def_zone)
+	if(!ishuman(M))
+		return
+	var/mob/living/carbon/human/H = M
 	if(user.zone_selected == "eyes")
-		user.visible_message("<span class='notice'>You show the paper to [M]. </span>", \
-			"<span class='notice'> [user] holds up a paper and shows it to [M]. </span>")
-		M.examinate(src)
+		user.visible_message("<span class='notice'> [user] holds up a paper and shows it to [H]. </span>",
+			"<span class='notice'>You show the paper to [H]. </span>")
```
Comment (farie82) [inline code/modules/paperwork/paper.dm:144]:
```suggestion
		user.visible_message("<span class='notice'>[user] holds up a paper and shows it to [H].</span>",
			"<span class='notice'>You show the paper to [H].</span>")
```
Weird extra space there after and before the sentence

File: code/modules/paperwork/paper.dm [hunk 4]
```diff
@@ -135,29 +135,28 @@
 		show_content(user, forcestars = 1)
 	return
 
-/obj/item/paper/attack(mob/living/carbon/M as mob, mob/living/carbon/user as mob)
+/obj/item/paper/attack(mob/living/carbon/M, mob/living/carbon/user, def_zone)
+	if(!ishuman(M))
+		return
```
Comment (farie82) [inline code/modules/paperwork/paper.dm:140]:
```suggestion
		return ..()
```

**Review by farie82 — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/paper.dm [hunk 5]
```diff
@@ -135,31 +135,29 @@
...
-					H.lip_style = null
-					H.update_body()
-	else
-		..()
+		if(H == user)
+			to_chat(user, "<span class='notice'>You wipe off your face with [src].</span>")
+		else
+			user.visible_message("<span class='warning'>[user] begins to wipe [H]'s face clean with \the [src].</span>",
+							 	 "<span class='notice'>You begin to wipe off [H]'s face.</span>")
+			if(!do_after(user, 1 SECONDS, target = H) && do_after(H, 1 SECONDS, FALSE)) // user needs to keep their active hand, H does not.
```
Comment (farie82) [inline code/modules/paperwork/paper.dm:153]:
```suggestion
			if(!do_after(user, 1 SECONDS, target = H) || !do_after(H, 1 SECONDS, FALSE)) // user needs to keep their active hand, H does not.
```
It is a negative check now. So if either failed then it should return.
Sidenote. A bit odd to have the double `do_after` here but it was already there so why not

File: code/modules/paperwork/paper.dm [hunk 6]
```diff
@@ -135,31 +135,29 @@
...
+			user.visible_message("<span class='warning'>[user] begins to wipe [H]'s face clean with \the [src].</span>",
+							 	 "<span class='notice'>You begin to wipe off [H]'s face.</span>")
+			if(!do_after(user, 1 SECONDS, target = H) || !do_after(H, 1 SECONDS, FALSE)) // user needs to keep their active hand, H does not.
+				return
+			user.visible_message("<span class='notice'>[user] wipes [H]'s face clean with \the [src].</span>",
+				"<span class='notice'>You wipe off [H]'s face.</span>")
+
+		H.lip_style = null
+		H.lip_color = null
+		H.update_body()
```
Comment (farie82) [inline code/modules/paperwork/paper.dm:160]:
```suggestion
		H.update_body()
	else
		return ..()
```
Else you can't bap people with the paper anymore

**Comment by AffectedArc07** [comment]
Please deal with the requested change within 7 days or this PR will be closed.

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**